.github/workflows/deploy-preview.yml

```
@@ -82,3 +92,9 @@
         env:
           VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
           VERCEL_PROJECT_ID: ${{ secrets.VERCEL_API_SERVER_PROJECT_ID }}
+
+  call-chromatic:
+    uses: ./.github/workflows/chromatic.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/chromatic.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)